recipe/build.sh

```diff
@@ -14,3 +14,13 @@ export MAKEFLAGS=""
 make
 # make run_tests  # Image mismatch issues.
 make install
+
+# Activate pyferret env vars.
+ACTIVATE_DIR=$PREFIX/etc/conda/activate.d
+DEACTIVATE_DIR=$PREFIX/etc/conda/deactivate.d
+mkdir -p $ACTIVATE_DIR
+mkdir -p $DEACTIVATE_DIR
+
+cp $RECIPE_DIR/scripts/activate.sh $ACTIVATE_DIR/pyferret-activate.sh
+cp $RECIPE_DIR/scripts/deactivate.sh $DEACTIVATE_DIR/pyferret-deactivate.sh
```

Comment:
@karlmsmith here is where the "conda magic" happens. Everything in these scripts will be execute when activating (or deactivating) a conda environment.